verticapy/datasets/datasets.py

```diff
@@ -24,15 +24,16 @@
 
 # VerticaPy Modules
 import verticapy, vertica_python
-from verticapy.utils._decorators import (
-    save_verticapy_logs,
-    check_minimum_version,
-)
-from verticapy import vDataFrame
+from verticapy._version import check_minimum_version
+from verticapy._utils._collect import save_verticapy_logs
+from verticapy.core.vdataframe import vDataFrame
```

Comment:
at one place I saw you do:
from verticapy.core.vdataframe.vdataframe import vDataFrame
i think better to be consistent

Reply:
done